.github/ISSUE_TEMPLATE/summit-topic-proposal.md

```diff
@@ -2,7 +2,7 @@
 name: Summit Session Proposal
 about: Submit a proposal for a session at the Collab Summit
 title: 'Session Proposal: <Project Name> <Session Title>'
-labels: Bilbao 2023, Collaborator Summit, Session Proposal

```

Comment:
I think we should remove the edition specific label, as the link on the Summit Announcement already redirects you to a new issue form with all correct labels applied.

Reply:
done